packages/performance/src/services/perf_logger.test.ts

```diff
@@ -121,10 +121,10 @@ describe('Performance Monitoring > perf_logger', () => {
       expect(addToQueueStub).not.to.be.called;
     });
 
-    it('Amount of customMetric is 32.', () => {
+    it('Ascertains that the max number of customMetric allowed is 32', () => {
```

Comment:
'ascertains'. Description starts with small letter (the whole describe and it descriptions should read as a sentence).